.gitignore

```diff
@@ -18,7 +18,7 @@ pip-wheel-metadata
 
 # PyTest
 .pytest_cache
-pytest.xml
+.pytest_results
 geckodriver.log
 
 # MyPy
```

Comment:
Would it be possible to add a filter as well for 

```
# Pyenv
.python-version
```

Reply:
Done 👍